NBitcoin/Crypto/Hashes.cs

```diff
@@ -18,24 +18,24 @@ namespace NBitcoin.Crypto
 {
 	public static class Hashes
 	{
-		#region Hash256
-		public static uint256 Hash256(byte[] data)
+		#region DoubleSHA256
+		public static uint256 DoubleSHA256(byte[] data)

```

Comment:
this is breaking API; you should leave the old methods there but with an `[Obsolete]` attribute

Reply:
Agreed...will change it, to be backward compatible